ICSharpCode.Decompiler/Semantics/Conversion.cs

```diff
@@ -76,6 +76,11 @@ public static Conversion EnumerationConversion(bool isImplicit, bool isLifted)
 		/// </summary>
 		public static readonly Conversion TryCast = new BuiltinConversion(false, 9);
 
+		/// <summary>
+		/// C# 7 throw expression being converted to an arbitrary type.
+		/// </summary>
+		public static readonly Conversion ThrowExpressionConversion = new BuiltinConversion(true, 10);
```

Comment:
Merge conflict: ThrowExpressionConversion and ImplicitInterpolatedStringConversion both have number 10?